guide/inheritance-wallet/onboarding-cosigners.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: guide
+title: Onboarding cosigners
+description: This page describes how Alice sets up the wallet in her app and activates it on her signing device.
+nav_order: 3
+parent: Inheritance wallet
+permalink: /guide/inheritance-wallet/onboarding-cosigners/
+main_classes: -no-top-padding
+image_base: /assets/images/guide/inheritance-wallet/onboarding-cosigners/
+images_bob-export-config:
+    - file: bob-export-config/onboard-alice-home
+      alt: Home screen showing three tasks that are associated with the wallet for the user to complete.
+      caption: A contextual task list helps users to complete all the necessary steps to safely use their new wallet.
+    - file: bob-export-config/onboard-alice-dialog
+      alt: Sharing dialog showing a QR code and sharing options.
+      caption: Bob exports the wallet configuration, so that Alice can set up the wallet on her phone.
+    - file: bob-export-config/onboard-alice-mark-complete
+      alt: Home screen showing three tasks that are associated with the wallet for the user to complete.
+      caption: After Alice has successfully set up the wallet on her end, Bob checks off the task. 
+    - file: bob-export-config/onboard-alice-completion-dialog
+      alt: Screen showing a confirmation dialog.
+      caption: Bob confirms that all cosigners have been onboarded. In our use case, Bob and Alice are the only cosigners.
+    - file: bob-export-config/onboard-alice-done
+      alt: Home screen showing two tasks associated with the wallet.
+      caption: The cosigner onboarding task is gone from the list and Bob can continue with the the next task.
+images_alice-import-config:
+    - file: alice-import-config/app-home-initial
+      alt: Screen showing an empty app home screen.
+      caption: Alice taps the plus button to start adding the wallet. 
+    - file: alice-import-config/add-wallet-sheet
+      alt: Screen showing different options for adding a wallet.
+      caption: She selects "Recover an existing wallet"
+    - file: alice-import-config/alice-import-config-options
+      alt: Screen showing different options for importing the wallet configuration.
+      caption: The easiest way to import the configuration is to scan the QR code from Bob's screen.
+    - file: alice-import-config/alice-scan-config
+      alt: Screen showing an open QR code scanner.
+      caption: Alice scans the QR code.
+    - file: alice-import-config/alice-review-wallet
```

Comment:
Should this title maybe also reflect the activity Alice is doing - "Onboarding as a co-signer"? The text could be something like "You are accepting an invite to become a co-signer on this wallet. It means that x, y and z..."